CONTRIBUTING.md

```diff
@@ -225,6 +226,23 @@ echo "<commit message>" | commitlint
 
 We don't enforce naming of PRs and branches, but we recommend you follow the same style. It can simply be one of your commit messages, just copy/paste it, e.g. `fix(readme): improve the readability and move sections`.
 
+<a name="-adding a dependency"></a>
+## Addind a dependency
+
+To add a depepency to DocArray you need to edit the `pyproject.toml` and add your dependency in the `[tool.poetry.dependencies]` section.
+Always overwrite poetry default version number (if you used `poetry add XYZ`):
+1- pick an appropirate version number. You should not nescarry pick the latest one but rather pick the old enough one that is still compatible
+2- use the `>=` notatation instead of the `~` to not lock upper limit
```

Comment:
```suggestion
- Pick an appropirate version number. You should not nescarry pick the latest one but rather pick the old enough one that is still compatible
- Use the `>=` notation instead of the `~` to not lock upper limit
```